src/pages/day2.js

```diff
@@ -1,8 +1,9 @@
 import { Link } from "react-router-dom"
 import { Code } from '../components/code/code'
-
+import { PageHeader } from '../components/pageHeader/pageHeader'
 export const Day2 = () => {
     return <div className="day2">
+        <PageHeader>CREATING THE BASIC BUILDING STONES OF THE INTERNET SITE</PageHeader>
```

Comment:
"CREATING THE BASIC BUILDING STONES OF THE INTERNET SITE" -> CREATING THE BASIC BUILDING STONES OF A REACT WEBSITE